src/components/POLine/POLineAgreementLines/POLineAgreementLinesContainer.js

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  Accordion,
+  Badge,
+  Loading,
+} from '@folio/stripes/components';
+import {
+  baseManifest,
+  LIMIT_MAX,
+  useShowCallout,
+} from '@folio/stripes-acq-components';
+
+import { AGREEMENT_LINES_API } from '../../Utils/api';
+import { ACCORDION_ID } from '../const';
+import POLineAgreementLines from './POLineAgreementLines';
+
+const POLineAgreementLinesContainer = ({ lineId, label, mutator }) => {
+  const [agreementLines, setAgreementLines] = useState();
+  const showCallout = useShowCallout();
+
+  useEffect(() => {
+    setAgreementLines();
+
+    mutator.agreementLines.GET({
+      params: {
+        filters: `poLines.poLineId==${lineId}`,
+        perPage: LIMIT_MAX,
```

Comment:
So basically here we can support Load more button. Could you please check with Dennis if we have to?

Reply:
ok. I'll ask him about it